### ISSUE TITLE
addNamedEntityText is not working in Chinese (zh)

### ISSUE THREAD

**Issue by thiefjim** [axa-group/nlp.js#172]
**Describe the bug**
I try to add entity with Chinese. 
Chinese entity is not good as English.

**To Reproduce**
1. No entities found.
```Code
const { NerManager } = require('node-nlp');

const manager = new NerManager({threshold:0.01 });
manager.addNamedEntityText(
    'place',
    '餐廳',
    ['zh'],
    ['餐館'],
);
(async () => {
    manager.findEntities(
        '找餐廳',
        'zh',
    ).then(entities => {
        console.log(entities);
    })
})();

```
2. Can find entities with low accuracy.
```Code
const { NerManager } = require('node-nlp');

const manager = new NerManager({threshold:0.01 });
manager.addNamedEntityText(
    'place',
    '餐廳',
    ['zh'],
    ['餐館','義大利餐廳','法國餐廳'],
);
(async () => {
    manager.findEntities(
        '找餐廳',
        'zh',
    ).then(entities => {
        console.log(entities);
    })
})();
```
3. Entity Found 0.5 accuracy
```Code
const { NerManager } = require('node-nlp');

const manager = new NerManager({threshold:0.01 });
manager.addNamedEntityText(
    'place',
    '餐廳',
    ['zh'],
    ['餐館'],
);
(async () => {
    manager.findEntities(
        '找 餐廳',
        'zh',
    ).then(entities => {
        console.log(entities);
    })
})();
```

**Expected behavior**
I have tried addNamedEntityText in [en](https://github.com/axa-group/nlp.js/blob/master/docs/ner-manager.md#enum-named-entities). It works. 
I would like to input 
```
manager.findEntities(
        '找餐廳',
        'zh',
    )
```
and the entities result can extract "餐廳".
How can improve the accuracy in Chinese ? Is this a bug?

**Screenshots**
1. Code 1 result
![image](https://user-images.githubusercontent.com/4970547/56496327-891eb180-652b-11e9-8543-26807fa59861.png)
2. Code 2 result
![image](https://user-images.githubusercontent.com/4970547/56496331-8f149280-652b-11e9-9e94-d8425a9f78ef.png)
3. Code 3 result
![image](https://user-images.githubusercontent.com/4970547/56496334-9340b000-652b-11e9-8eaa-5f90583e7098.png)

**Desktop (please complete the following information):**
 - OS: Windows Node v8.15.1
 - test by Node.js
 - Version 3.0.2

**Comment by jesus-seijas-sp** [comment]
Hello @thiefjim , bug confirmed. Fixed in version 3.0.3. Can you confirm that it works now?

 It was caused because when searching entities, an optimization is done splitting by whitespaces, and it was not splitting chinese characters properly.
In the other hand, the name of each option is not automatically added as a text of the option, so in this example the accuracy is 0.5:

```javascript
const { NerManager } = require('node-nlp');

const manager = new NerManager({threshold:0.01 });
manager.addNamedEntityText(
    'place',
    '餐廳',
    ['zh'],
    ['餐館','義大利餐廳','法國餐廳'],
);
(async () => {
    manager.findEntities(
        '找餐廳',
        'zh',
    ).then(entities => {
        console.log(entities);
    })
})();
```

To have accuracy 1, replace this part:
```javascript
manager.addNamedEntityText(
    'place',
    '餐廳',
    ['zh'],
    ['餐廳', '餐館','義大利餐廳','法國餐廳'],
);
```

**Comment by thiefjim** [comment]
To @jesus-seijas-sp 
Fixed Confirm. 
I have download the node-nlp 3.0.2.
The result is shown as follows
![image](https://user-images.githubusercontent.com/4970547/56544602-98dbdb80-65a7-11e9-8307-2ae5b67584ff.png)

Thanks for helping.
